pkg/agent/config/config.go

```diff
@@ -293,7 +293,7 @@ func locateOrGenerateResolvConf(envInfo *cmds.Agent) string {
 	return tmpConf
 }
 
-func get(envInfo *cmds.Agent, proxy proxy.Proxy) (*config.Node, error) {
+func get(ctx context.Context, envInfo *cmds.Agent, proxy proxy.Proxy) (*config.Node, error) {
```

Comment:
What are we getting with the addition of the context value?

Reply:
The legacy net.Listen predates contexts, but ListenConfig.Listen was added later and wants the context plumbed through. I could have just gone with context.Background() but actually passing it through seemed like the correct thing to do.

Reply:
Figured but also figured I'd ask.